src/components/Article/index.tsx

```diff
@@ -55,6 +57,7 @@ export default class Article extends React.Component<ArticleProps> {
     return (
       <article
         className={css`
+          ${style};
```

Comment:
Also let's put the style at the end – that way it can [override the default styles](https://emotion.sh/docs/composition).

Reply:
k think i rewrote all of the components to do this